config.php

```diff
@@ -9,7 +9,7 @@
 /*
  * Change this for the URL of your repository.
  */
-define('PRISMIC_URL', 'https://your-repo-name.prismic.io/api');
+define('PRISMIC_URL', 'https://your-repo-name.prismic.io/api/v2');
```

Comment:
Need to remember to mention this line change in the actual user guide, it's the only required modification in that text to reflect the new code